Automattic/pocket-casts-android
# Discover sorts categories alphabetically

## Description
Fixing an issue where the category list inside discover is sorted based on the english strings. If a user has their device in a different language then the list is not sorted alphabetically.

Fixes #673

## Testing Instructions
1. Fresh install of the app.
2. Start the app.
3. Navigate to the discover page with system settings in English.
4. Notice that the categories listed under "Browse By Category" are sorted alphabetically.
5. Set your system language to another language e.g. French.
6. Navigate to the discover page.
7. Notice that the categories listed under "Bowser By Category" are still sorted alphabetically.

## Screenshots or Screencast 
![PocketcaseEng](https://user-images.githubusercontent.com/1049216/236584732-5f9e8bb2-e635-4ff5-b598-76006753cd13.png)
![PocketcastsFr](https://user-images.githubusercontent.com/1049216/236584735-e44fb007-15fc-42e0-8e20-d9b538e976c0.png)


## Checklist
 
#### I have tested any UI changes...
